src/responsive-element.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+var assignBreakpointClasses = function(parent, breakpoints) {
+	var parentWidth = parent.offsetWidth;
+		
+	for (var x=0, l=breakpoints.length; x < l; x++) {
+		if (parentWidth < breakpoints[x].width) {			
+			parent.className += ' ' + breakpoints[x].class;
+		}
+	}
+};
+
+module.exports = function(parent, breakpoints) { // TO DO improve bp handling to recheck on resize (but effeciently...) into own module

```

Comment:
The plan is to setup an event (of some sort, that performs well) to recheck the breakpoints when the element changes size.  But you're right that it could just return the function for now.  However, knowing I expect this to change significantly soon I'll leave it be.